server/api/messageCenter/getTopic.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response } from 'express'
+import Requests from '@server/utils/requests'
+import { MessageCenterController } from '@server/controller/messageCenter'
+import { CountryIso } from '@core/country'
+
+export const getTopic = async (req: Request, res: Response) => {
+  try {
+    const { countryIso, assessmentName, cycleName, key } = req.query
+
+    const topic = await MessageCenterController.getTopic({
+      countryIso: countryIso as CountryIso,
+      assessmentName: String(assessmentName),
+      cycleName: String(cycleName),
+      key: String(key),
```

Comment:
As previous, I think it's better to use `<Record<string, string>>req.query` or add helper `Requests.getParams()`